package.json

```diff
@@ -1,19 +1,20 @@
 {
   "dependencies": {
-    "@angular/common": "2.0.0-rc.3",
-    "@angular/compiler": "2.0.0-rc.3",
-    "@angular/core": "2.0.0-rc.3",
-    "@angular/platform-browser": "2.0.0-rc.3",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.3",
-    "@angular/http": "2.0.0-rc.3",
+    "@angular/common": "^2.0.0-rc.3",
```

Comment:
Do NOT do this! This is really dangerous!
Adding ^ to the version code, means that npm will use this version OR any version above this. Since we are working with early beta versions of both ionic2 and angular2, any new version, could introduce breaking changes. Therefore, it is really important, that everyone developing and using this app, uses the same versions. Otherwise, it will be quite certain, that someone running `npm install` in a year or so, won't get a single line of code to work!